tmail-backend/integration-tests/jmap/distributed-jmap-integration-tests/src/test/java/com/linagora/tmail/james/DistributedMailboxSetMethodTest.java

```diff
@@ -54,6 +55,7 @@ public class DistributedMailboxSetMethodTest implements MailboxSetMethodContract
         .extension(new DockerOpenSearchExtension())
         .extension(new CassandraExtension())
         .extension(new RabbitMQExtension())
+        .extension(new RedisExtension())
```

Comment:
Please confirm that Redis is not mandatory if I choose the pure rabbitmq implem.

I want to be able to start tmail without redis

This had been VERY valuable on CNB...

Reply:
cf https://github.com/linagora/tmail-backend/pull/943/commits/1a7614b9e3f834280c32028721c635ddaca73e12